fs/walk.py

```diff
@@ -398,19 +416,27 @@ def _walk_breadth(
         queue = deque([path])
         push = queue.appendleft
         pop = queue.pop
-        depth = self._calculate_depth(path)
+
+        _combine = combine
```

Comment:
Micro-optimizations. Locals are faster than an attribute on self, which requires a dict lookup.

TBH the difference may only be noticeable on benchmarks. In the real world where work is done on the fs being walked, these speed-ups will be negligible.